ui/src/heap/HeapBlock.tsx

```diff
@@ -100,7 +105,7 @@ function TopBar({
           )}
         </div>
         {canEdit && (
-          <div className="relative hidden group-hover:block">
+          <div className={longPress ? 'relative' : ' hidden group-hover:block'}>
```

Comment:
seems like you may still want relative when not longpressed?

Reply:
Yeah, you're right.